packages/web/src/scss/settings/_utilities.scss

```diff
@@ -1,25 +1,49 @@
 @use 'sass:map';
 @use '@tokens' as tokens;
+@use '../tools/utilities' as tools-utilities;
+
+$accent-background-scope-list: background-basic, background-subtle;
+$accent-border-color-scope-list: border-basic, border-subtle;
+$accent-text-color-scope-list: content-basic, content-subtle;
+
+$accent-prefix: 'accent-';
+$accent-tokens-key: 'accent-colors';
 
 $utilities: (
     'background-color': (
         responsive: false,
         property: background-color,
         class: bg,
-        values: (
-            primary: tokens.$background-primary,
-            secondary: tokens.$background-secondary,
-            tertiary: tokens.$background-tertiary,
-        ),
+        values: map.merge(
+                tools-utilities.generate-token-map(
+                    $accent-tokens-key,
+                    $accent-prefix,
+                    'background',
+                    $accent-background-scope-list
```

Comment:
I prefer to list these vars here directly without another reference. The var is used only once, and you already defined `background` here one line above, so why not list everything background-related in one place?

Reply:
Forgotten. Removed. 👍🏻